src/libcollections/vec.rs

```diff
@@ -1301,6 +1301,41 @@ impl<T: Clone> Vec<T> {
     }
 }
 
+impl<T: Default> Vec<T> {
+    /// Resizes the `Vec` in-place so that `len()` is equal to `new_len`.
+    ///
+    /// If `new_len` is greater than `len()`, the `Vec` is extended by the
+    /// difference, with each additional slot filled with `Default::default()`.
+    /// If `new_len` is less than `len()`, the `Vec` is simply truncated.
```

Comment:
Maybe add a note "See also `Vec::resize`, which uses clone."? I don't know what policy is on that, though (cc @rust-lang/docs)